mne/viz/_brain/_brain.py

```diff
@@ -2654,24 +2653,32 @@ def show(self):
         """Display the window."""
         self._renderer.show()
 
-    def show_view(self, view=None, roll=None, distance=None, row=0, col=0,
+    def show_view(self, view=None, azimuth=None, elevation=None, roll=None,
```

Comment:
do not change order of arguments, you will break existing working code

Reply:
Better as keyword arguments in `view`? That might be more backward compatible, then just `roll` and `distance` are depreciated

Reply:
I think I actually prefer the way you have it now (other than order of arguments, new ones need to be at the end), otherwise people end up writing stuff like `show_view(dict(azimuth=..., elevation=..., focalpoint=...))` instead of just `show_view(azimuth=..., elevation=..., focalpoint=...)` so `dict` is just an extra unnecessary hoop to jump through.

Basically we need to support `view` as dict rather than `None | str` (what we're deprecating toward) until 0.25 comes out